### ISSUE TITLE
icons should be svg

### ISSUE THREAD

**Issue by bartvde** [boundlessgeo/qgis-geogiglight-plugin#99]
So they are better visible on hidpi screens

![selection_015](https://cloud.githubusercontent.com/assets/319678/21925928/e7046b12-d97f-11e6-8bb9-b11922c4eec3.png)

**Comment by SrNetoChan** [comment]
Who design those? Some I recognize from QGIS.

I can try to create SVG versions for it.

**Comment by SrNetoChan** [comment]
@bartvde the icons were converted but I don't have a hidpi screen to check if it works. Do you?

**Comment by bartvde** [comment]
I do, I will check tomorrow

**Comment by bartvde** [comment]
this is what I get now:

![selection_226](https://cloud.githubusercontent.com/assets/319678/25167894/b14bee5c-24e1-11e7-84b5-6e63ddb37f2a.png)

**Comment by SrNetoChan** [comment]
Is it still too small? I guess that the icon size must be set in the plugin. @volaya any ideas?

**Comment by volaya** [comment]
mmm, i will see, but i think Qt should adapt the size to the screen resolution. The size of the icon is not set, we are just specifying the icon file, so that is done by Qt (apparenty not correctly). I will check

**Comment by elpaso** [comment]
@volaya Qt5 should handle this correctly, but I couldn't find a better way than rendering svg to `QPixMap` for Qt4 such as in  https://github.com/boundlessgeo/qgis-basemaps-plugin/blob/master/boundlessbasemaps/gui/setupwizard.py#L75

**Comment by elpaso** [comment]
Sorry, the link was for PNG, this is for SVG  https://github.com/boundlessgeo/qgis-basemaps-plugin/blob/master/boundlessbasemaps/gui/setupwizard.py#L439 but the logic is the same

**Comment by volaya** [comment]
I tried to use that strategy, but it seems i cannot put a larger icon in the toolbar

I have now

QIcon(QPixmap(icon.pixmap(size)))

if size is smaller, i get tiny icons, so it works. If I set the size to a larger thing (let's say 100x100), I still get the regular size icon.

I have tried adding also self.navigatorToolbar.setIconSize(size), but it doesnt work. Qt docs say that wont scale smaller icons, but looks like even if the icon is already large (explicitely scaled when creating the new icon), it will put it at the normal size.

Any ideas?
